charts/bindplane/templates/bindplane.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 spec:
-  {{- if eq (include "bindplane.deployment_type" .) "StatefulSet" }}
+  {{- if gt .Values.replicas 0 }}
```

Comment:
We do not need to check if StatefulSet. If it is a statefulset, and replicas is not set, it will default to 1 replica.

What we want to do is check if values.replicas is greater than 0, if it is, set it explicitly.